test/e2e/node/kubelet.go

```diff
@@ -435,8 +435,9 @@ var _ = SIGDescribe("kubelet", func() {
 					By("Stop the NFS server")
 					stopNfsServer(nfsServerPod)
 
-					By("Delete the pod mounted to the NFS volume")
-					framework.ExpectNoError(framework.DeletePodWithWait(f, c, pod), "Failed to delete pod ", pod.Name)
+					By("Delete the pod mounted to the NFS volume -- expect failure")
+					err := framework.DeletePodWithWait(f, c, pod)
+					Expect(err).To(HaveOccurred())
```

Comment:
See above re: using one pattern or the other.

Reply:
ack